gapis/api/sync/sync.go

```diff
@@ -81,7 +81,9 @@ func MutationCmdsFor(ctx context.Context, c *path.Capture, cmds []api.Cmd, id ap
 			if err != nil {
 				return nil, err
 			}
-			terminators = append(terminators, term)
+			if term != nil {
```

Comment:
You probably want to insert the EarlyTerminator in this case (or return it from GetTerminator), otherwise all commands for this API will be kept even if you only want a subset of the commands.